### PR TITLE
Binary, base64 and none encoding

### PR DESCRIPTION
**Pull request by christophermancini**
To address #122 (CLIENTS-698) (CLIENTS-918), I consumed PR #123 (thanks @benscott!) and expanded upon it to offer better support overall when working with binary and base64 objects on the HTTP interface.

### CODE REVIEW

**Comment by christophermancini** [comment]
@benscott & @mablo can you review to determine if this PR looks good and meets your needs?

**Comment by lukebakken** [comment]
Does the Travis build depend on #132 for this PR?

**Comment by lukebakken** [comment]
Other than unhappy Travis, :+1:

**Comment by christophermancini** [comment]
Yes, #132 switches over to using PackageCloud as the source of Riak. Travis's version of Riak as a service is outdated and does not contain Preflist support. :/

File: src/Riak/Api/Http/Translator/ObjectResponse.php [hunk 1]
```diff
@@ -90,6 +90,6 @@ public function parseObject($response, $headers = [])
         $contentType = !empty($headers[Http::CONTENT_TYPE_KEY]) ? $headers[Http::CONTENT_TYPE_KEY] : '';
         $data = $this->command->getDecodedData($response, $contentType);
 
-        return new Object($data, $headers);
+        return (new Object($data, $headers))->setRawData($response);
```
Comment (alexmoore) [inline src/Riak/Api/Http/Translator/ObjectResponse.php]: So we always `setRawData` now?

Comment (christophermancini) [inline src/Riak/Api/Http/Translator/ObjectResponse.php]: Yea, I am doing it for debugging purposes.

**Pull request merged**